code/modules/research/designs/weapon_designs.dm

```diff
@@ -190,7 +190,7 @@
 	build_path = /obj/item/ammo_box/wt550/wttx
 	category = list("Weapons")
 
-/datum/design/mag_laser
+/datum/design/laser_rifle_magazine
```

Comment:
im pretty sure design/balance wants these cargo order only, unsure about this fix

Reply:
can you shoot them a ping in devchat to confirm this? I'm not gonna deal with indecisiveness for this

Reply:
They think its fine

Reply:
👍